frontend/src/components/ExportChat/ExportChatMessage.tsx

```diff
@@ -71,6 +72,7 @@ function getMessageStyle(type: CHAT_MESSAGE_TYPE) {
 			return styles.chatBoxInfo;
 		case CHAT_MESSAGE_TYPE.USER:
 			return styles.chatBoxMessage;
+		case CHAT_MESSAGE_TYPE.BOT_BLOCKED:
```

Comment:
I don't quite get what's going on with these empty case statements. How does it behave? Could we write it explicitly?

Reply:
Empty case statements just drop down to the next case statement. It's fairly common to see a "fallthrough" comment in these cases, so we could do that.

```typescript
case CHAT_MESSAGE_TYPE.BOT_BLOCKED:
        // fallthrough
case CHAT_MESSAGE_TYPE.BOT:
	return styles.chatBoxMessageBot;
```

Reply:
so 
```typescript
case CHAT_MESSAGE_TYPE.BOT_BLOCKED:
        // fallthrough
case CHAT_MESSAGE_TYPE.BOT:
	return styles.chatBoxMessageBot;
```
is the same as
```typescript
case CHAT_MESSAGE_TYPE.BOT_BLOCKED || CHAT_MESSAGE_TYPE.BOT:
	return styles.chatBoxMessageBot;
```
?

I guess it's more aesthetically pleasing this way. If this is a common thing to do then I'm happy with it!

Reply:
Sort of, except the latter doesn't quite work. It's sort of the same as doing 
`if (type === (CHAT_MESSAGE_TYPE.BOT_BLOCKED || CHAT_MESSAGE_TYPE.BOT))`
which would be like doing
`if (type === false)`
which isn't what you want.

With switch statements, if you want the same result for multiple values, yes the former is the way to do it 😃

Reply:
Ahhh, right! Thanks for educating me :)